tests/smoke/support/temp_project.rs

```diff
@@ -157,11 +157,11 @@ fn volta_tools_dir(root: PathBuf) -> PathBuf {
 fn inventory_dir(root: PathBuf) -> PathBuf {
     volta_tools_dir(root).join("inventory")
 }
-fn user_toolchain_dir(root: PathBuf) -> PathBuf {
-    volta_tools_dir(root).join("user")
+fn default_toolchain_dir(root: PathBuf) -> PathBuf {
+    volta_tools_dir(root).join("default")
 }
-fn user_dir(root: PathBuf) -> PathBuf {
-    volta_tools_dir(root).join("user")
+fn default_dir(root: PathBuf) -> PathBuf {
+    volta_tools_dir(root).join("default")
```

Comment:
Interestingly, this function appears to be an exact duplicate of the above one with a different name. Perhaps we should just remove it and migrate everything to use `default_toolchain_dir`

Reply:
removed `user_dir` and migrated to `default_toolchain_dir`